src/Configuration/Configuration.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Configuration;
+
+use Illuminate\Support\Arr;
+use MyParcelCom\Integration\Configuration\Properties\PropertyCollection;
+use MyParcelCom\Integration\ProvidesJsonAPI;
+
+class Configuration implements ProvidesJsonAPI
+{
+    /**
+     * @param PropertyCollection $properties
+     */
+    public function __construct(private readonly PropertyCollection $properties)
+    {
+    }
+
+    public function transformToJsonApiArray(): array
+    {
+        return [
+            '$schema'              => 'http://json-schema.org/draft-04/schema#',
+            'additionalProperties' => false,
+            'required'             => [], // ???
```

Comment:
I wasn't 100% certain but I think it is unlikely that any of the configuration form fields will be required, so shall I remove this?
```suggestion
```

Reply:
I think we should keep it!

Reply:
Idea: we can map all property names that have property isRequired=true